src/main/java/gvs/business/logic/graph/Session.java

```diff
@@ -233,20 +256,26 @@ private boolean validIndex(int i) {
    * @param targetGraph
    *          target graph
    */
-  private void takeOverPreviousVertexPositions(Graph sourceGraph,
+  private boolean takeOverPreviousVertexPositions(Graph sourceGraph,
```

Comment:
name der Methode und Rückgabewert passt nicht gut zusammen. würde entweder name anpassen oder in javadocs beschreiben, was dieser boolean aussagt